sample/src/main/java/com/google/accompanist/sample/insets/EdgeToEdgeLazyColumn.kt

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+@file:Suppress("DEPRECATION")
```

Comment:
Please leave a comment on the sample uses of this with something like "to be replaced in a later PR to test the migration"

Reply:
I wasn't planning to publish those in-place to the repo, although I have a branch with those changes here working off of the migrations:
https://github.com/google/accompanist/tree/av/deprecated-insets-replace-samples